README.md

```diff
@@ -1,52 +1,51 @@
-[![Build Status](https://travis-ci.org/Kinvey/js-sdk.svg?branch=refactor%2FMLIBZ-2587_Improve_SDK)](https://travis-ci.org/Kinvey/js-sdk)
+<p align="left">
+  <a href="https://www.progress.com/kinvey" style="display: inline-block;">
+    <img src="logo-progresskinvey.png">
+  </a>
+</p>
 
-# Overview
+<p align="left">
+  <a href="https://cirrus-ci.com/github/Kinvey/js-sdk"><img alt="Build Status" src="https://api.cirrus-ci.com/github/Kinvey/js-sdk.svg"></a>
+  <a href="https://conventionalcommits.org"><img alt="Conventional Commits" src="https://img.shields.io/badge/Conventional%20Commits-1.0.0-yellow.svg"></a>
+</p>
```

Comment:
Should use macros here for badges, same as what the other SDK READMEs do.

Reply:
You are right. I meant to change it back. I was trying a different format so that I could center them in the README to see what it would look like.

Reply:
I have fixed this with d8ab435